src/sidebar/services/load-annotations.js

```diff
@@ -16,53 +16,73 @@ export default function loadAnnotationsService(
   let searchClient = null;
 
   /**
-   * Load annotations for all URIs and groupId.
+   * Load all annotations that match `options` criteria
    *
-   * @param {string[]} uris
-   * @param {string} groupId
+   * @param {Object} options
+   *   @param {string} options.groupId
+   *   @param {string[]} [options.uris]
    */
-  function load(uris, groupId) {
+  function load(options) {
+    const { uris } = options;
     store.removeAnnotations(store.savedAnnotations());
 
     // Cancel previously running search client.
     if (searchClient) {
       searchClient.cancel();
     }
 
-    if (uris.length > 0) {
-      searchAndLoad(uris, groupId);
-
+    if (uris && uris.length > 0) {
       streamFilter.resetFilter().addClause('/uri', 'one_of', uris);
       streamer.setConfig('filter', { filter: streamFilter.getFilter() });
     }
+    searchAndLoad(options);
   }
 
-  function searchAndLoad(uris, groupId) {
+  /**
+   * @param {Object} options
+   *   @param {string[]} [options.uris]
+   *   @param {string} options.groupId
```

Comment:
I would order the parameters in the same order as the above function:
```suggestion
   *   @param {string} options.groupId
   *   @param {string[]} [options.uris]
```

Reply:
As in last comment, this function is going away...